packaging_apps_git_fr.md

```diff
@@ -1,72 +1,72 @@
 # Comment utiliser Git pour packager les applications
 
-Git... Notre cher Git bien-aimé, que l'on peut aussi décrire comme "Goddamn Idiotic Truckload of sh*t", selon Linus.  
+Git... Notre cher Git bien-aimé, que l'on peut aussi décrire comme "Goddamn Idiotic Truckload of sh*t"  ( P *t**n de camion de m*rde), selon Linus.
```

Comment:
That's a literal translation, and as I was quoting Linus Torvalds, so I kept the original version.
To translate that would looks like... "Un stupide putain gros tas de merde" Does it really worth it ?

Reply:
yes, Seems to me it keeps the spirit of the quote.

Reply:
I guess the original quote is not so obvious for non English speakers.
So, if you think it really worth it, why not.
But, do not censor more than the original quote. A censured quote from Linus isn't anymore from him !